// Enable generation of Protocol Buffers RPC services.
option cc_generic_services = true;
option java_generic_services = true;
option py_generic_services = true;

// Defining a print service.

// Define a request.

message PositionInformationTransmit {
  required string trainID = 1;
  required string position = 2;
  optional int32 status = 3;
}

// Define a response

message PositionInformationReceive {
  required string serverName = 2;
}

// Define the service.

service PositionInformationService
{
  rpc PositionInformation (PositionInformationTransmit) returns (PositionInformationReceive);
}

